### ISSUE TITLE
Is this project alive?

### ISSUE THREAD

**Issue by d47081** [kr1sp1n/awesome-gemini#226]
I've created few extensions like [ggemtext](https://github.com/YGGverse/ggemtext) and [ggemini](https://github.com/YGGverse/ggemini) + rewritten [Yoda](https://github.com/YGGverse/Yoda) in Rust, but previous PRs still pending

Just worried about conflicts when I send new commits over previous one, I want to cleanup some offline services also

**Comment by mathiscode** [comment]
Since this repository appears unmaintained, I've created a new fork and merged in the pending PRs.

https://github.com/mathiscode/awesome-gemini

I'll be doing cleanup on dead links soon, and if you want to add new links or help cleanup just create a new PR on that repo - my career depends on me accessing Github frequently, so I won't miss any PRs or issues and they won't stack up.

**Comment by mathiscode** [comment]
Yeah, I made a PR on this repo myself and knew it would probably never get merged. The new fork has your changes and if you need to add anymore they'll get merged quickly.

**Comment by kr1sp1n** [comment]
@mathiscode @d47081 Hi 👋 Thank you that you care about this repo. Sometimes I sleep very long (1y or even longer) and when I wake up a lot of things happened while I was away. I merged some very old PRs and started to maintain it further.
Have a nice weekend.

**Comment by mathiscode** [comment]
No worries, you have no obligation to maintain - everyone has a life to live! I'm happy to help out, I'll keep an eye on PRs and help you stay caught up 💪

**Comment by kr1sp1n** [comment]
@mathiscode Thanks for your gentle words and your offer to help.

**Comment by flexibeast** [comment]
@kr1sp1n: Ah okay, wow. i can sort of relate - for over a year now i've been dealing with severe chronic pain (on top of my preexisting chronic conditions) which often leaves me completely debilitated / unable to move. Which has left me with some fresh perspectives on things. 🙂 Thanks for your continued work on this repo! i've not been as active in geminispace as i'd like recently, due to the above, but hopefully i can spend more time there soon ....

@mathiscode: Thanks for your continued work also. 🙂
